### DISCUSSION TITLE
Has anyone used Alesis Multimix 4 to set up Jamulus?

### DISCUSSION THREAD

**Discussion by klepto84** [jamulussoftware/jamulus#1046] [Technical Support & Help] [answered]
We are a 4 person group using Jamulus and loving it!  Thank you to everyone who developed and supports this amazing program! Three members are using Focusrite Scarletts successfully.  The last member is trying to use the Alesis Multimix 4 and we have been unable to make it work.  I would appreciate any suggestions.  Been Googling like crazy and not able to find any helpful resource.
Thanks in advance.

**Comment by ann0see** [comment]
I don't know this device, but could you give some more information? What doesn't work? Do you have an error message?

Please keep in mind, that the device must support 48kHz sample rate.

**Reply by gibbsp1** [reply]
Did you ever get this working?

**Comment by dcorson-ticino-com** [comment, marked as answer]
I see that the Multimix 4 should support 48k.
You don't say what operating system you use.
If you have a PC you will need an ASIO driver, but I don't see any mention of one.
I would be looking for their ASIO driver.
If not, it may work with ASIO4ALL, but I wouldn't count on it.
